src/Command/BakeMigrationDiffCommand.php

```diff
@@ -506,7 +506,7 @@ protected function getDumpSchema(Arguments $args)
             $inputArgs['--plugin'] = $args->getOption('plugin');
         }
 
-        $className = '\Migrations\Command\Dump';
+        $className = '\Migrations\Command\Phinx\Dump';
```

Comment:
The leading \ should not be necessary
Personally, I would find it cleaner to use ::class syntax here.